soymsg/pomsg/pomsg.go

```diff
@@ -26,19 +27,37 @@ type FileOpener interface {
 }
 
 // Load returns a soymsg.Provider that takes its translations by passing in the
-// specifified locales to the given PoFileProvider.
+// specified locales to the given PoFileProvider.
 //
-// TODO: Fallbacks between <lang> and <lang>_<territory>
+// Supports fallbacks for when a given locale does not exist, as long as the fallback files are in
+// canonical form.
 func Load(opener FileOpener, locales []string) (soymsg.Provider, error) {
 	var prov = provider{make(map[string]soymsg.Bundle)}
 	for _, locale := range locales {
 		r, err := opener.Open(locale)
 		if err != nil {
 			return nil, err
 		} else if r == nil {
-			continue
+			localeTag, err := language.Parse(locale)
```

Comment:
I personally think it'd be clearer if this if statement was just another function called "handleFallback" or something.

Reply:
acking because I wound up removing this